### ISSUE TITLE
Old version works in Waterfox, but newer ones do not.

### ISSUE THREAD

**Issue by PixelEater** [teddy-gustiaux/default-bookmark-folder#8]
I'm not sure if you would be able to do anything about this, but in the latest release of Waterfox (56.0.1 (64-bit) this add-on seemingly does nothing. I found that actually version 1.0 successfully did change the default bookmarks folder, but I'm really after that add to top feature as well. Unfortunately Add Bookmark Here isn't working either. I wonder if you can see any way to restore the functionality since technically it did work at one point. I think true support for 56.* is a good goal at this time. I'm not positive if Waterfox could be basing something off the 54.* ESR and that could be causing it, but it does use the newer style bookmarks introduced in 56.

**Comment by teddy-gustiaux** [comment]
Thank you for reporting this issue! And for making me discover another fork of Firefox. 😃

I have indeed been able to verify that the built-in bookmarking features of the add-on do not work on the latest version of Waterfox, as well as with version 56.x of Firefox. The quick bookmarking features work fine. I have also found the origin of the problem: I have been using a property that is only available since the version 57 of Firefox. I am going to fix this right away and publish a fixed version of the add-on!
